.github/workflows/awie.yml

```
@@ -32,6 +32,8 @@ jobs:
   package:
     needs: [get-version]
 
+    if: ${{ contains(fromJson('["testing", "unstable"]'), needs.get-version.outputs.stability) }}
```

to apply everywhere :)
```suggestion
    if: ${{ needs.get-version.outputs.stability != 'stable' }}
```